apps/filecoin-site/src/app/_components/Footer/Footer.tsx

```diff
@@ -0,0 +1,68 @@
+import { SmartTextLink } from '@filecoin-foundation/ui/TextLink/SmartTextLink'
+
+import { footerNavigationItems, legalLinks } from '@/constants/navigation'
+import { BASE_DOMAIN } from '@/constants/siteMetadata'
+
+import { NewsletterForm } from '@/components/Footer/NewsletterForm'
+import { LogoFull } from '@/components/LogoFull'
+
+import { LogoFilecoinFoundation } from '@/_components/Footer/LogoFilecoinFoundation'
```

Comment:
@CharlyMartin - I think I saw a comment from you about this. Do we prefer this? I thought when files are local, it's nice to show?

```suggestion
import { LogoFilecoinFoundation } from '../LogoFilecoinFoundation'
```

Reply:
Small preference over `@/components/Footer`, but I can live with both :)